apisix/plugins/log-rotate.lua

```diff
@@ -85,7 +85,7 @@ local function get_last_index(str, key)
     return n
 end
 
-
+-- get the log directory path 
```

Comment:
The functions needs to be separated by two blank lines

Reply:
Done